### DISCUSSION TITLE
Set OLU_R22 OBX Result Value

### DISCUSSION THREAD

**Discussion by rjahrich** [nHapiNET/nHapi#406] [Q&A]
Hello,

I wonder if someone can help, we are using OLU_R22 messages and trying to create an OBX segment, creating the OBX is fine but I am struggling to set a value into OBX.5 (Result Value) 

We are using V25 models and the latest version of NHAPI from Nuget, if someone has an example on how to do this I would be greatful. 

Cheers

**Comment by milkshakeuk** [comment]
Hi maybe have a look at these tests... might help if not let me know.

https://github.com/nHapiNET/nHapi/blob/master/tests/NHapi.NUnit/Parser/PipeParserV25Tests.cs#L125
